tests/h/search/index_test.py

```diff
@@ -149,3 +270,18 @@ def __init__(self, annotation_ids):
     def __eq__(self, search_response):
         ids = [search_result.meta["id"] for search_result in search_response]
         return ids == self.annotation_ids
+
+
+@pytest.fixture
+def index(es_client, pyramid_request):
+    def _index(*annotations):
+        """Index the given annotation(s) into Elasticsearch."""
+        for annotation in annotations:
+            h.search.index.index(es_client, annotation, pyramid_request)
+        es_client.conn.indices.refresh(index=es_client.index)
+    return _index
```

Comment:
This index fixture has been moved to `tests/h/search/conftest.py](https://github.com/hypothesis/h/blob/master/tests/h/search/conftest.py) now, so you don't need it here anymore. (You'll need to rebase.)